README.adoc

```diff
@@ -102,8 +102,8 @@ Create the inventory manager class in the `src/main/java/io/openliberty/guides/i
 include::finish/src/main/java/io/openliberty/guides/inventory/InventoryManager.java[tags=**;!copyright;]
 ----
 
-This bean contains two simple functions. The `get()` function is for retrieving entries from calling the `system` service,
-and then add it to the inventory list.
+This bean contains three simple functions. The `get()` function is for retrieving entries from calling the `system` service.
+The `add()` function adds an entry to the inventory.
```

Comment:
Can you keep the format the same? ie. write "The add() function is for adding entries to the inventory"

Reply:
Ok, fixed.